tests/fsharp/core/signedtests/build.bat

```diff
@@ -0,0 +1,265 @@
+@if "%_echo%"=="" echo off
```

Comment:
`coreclr` is another branch, can we remove it from `master` at least? 
i didnt write the cleanup pr or opened the issue because i was thinking something like "we use it for internal build server" :smile: so i was waiting